svelte-cloudinary/src/lib/components/CldImage.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
+	import { getTransformations } from '@cloudinary-util/util';
+	import { transformationPlugins } from '@cloudinary-util/url-loader';
+	import type { ImageOptions } from '@cloudinary-util/url-loader';
+	import { Image } from '@unpic/svelte';
+
+	import { getCldImageUrl } from '$lib/helpers/getCldImageUrl.js';
```

Comment:
just for the context of why i had it `helpers` as a top level directory is they're 1st-class exports, at the same level as a component, as opposed to lib which is typically more internal functions/modules

Reply:
gotcha, wondering how to handle that here, the "convention" on SvelteKit is to use `$lib` to  access common components and utility modules

Reply:
oh i guess since everything is in lib, even the component, that makes sense. my point was mainly helpers and components being on the same level